From c597d61a615091f3d7176fa0cb3efc56b3dbda66 Mon Sep 17 00:00:00 2001
From: Michael Wright <michaelwr@google.com>
Date: Wed, 22 Aug 2018 13:49:32 +0100
Subject: [PATCH 22/48] Fix a typo when checking a touch is within the physical
 frame.

Bug: 112876936
Test: enable notchless simulation, try seascape apps, try swiping down
Change-Id: I8a155e16c19c474e605cb5a8d938e04461646027
---
 services/inputflinger/InputReader.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/inputflinger/InputReader.cpp b/services/inputflinger/InputReader.cpp
index 4c4786fea..0b7e85045 100644
--- a/services/inputflinger/InputReader.cpp
+++ b/services/inputflinger/InputReader.cpp
@@ -6547,7 +6547,7 @@ void TouchInputMapper::cancelTouch(nsecs_t when) {
 
 bool TouchInputMapper::isPointInsideSurface(int32_t x, int32_t y) {
     const float scaledX = x * mXScale;
-    const float scaledY = x * mYScale;
+    const float scaledY = y * mYScale;
     return x >= mRawPointerAxes.x.minValue && x <= mRawPointerAxes.x.maxValue
             && scaledX >= mPhysicalLeft && scaledX <= mPhysicalLeft + mPhysicalWidth
             && y >= mRawPointerAxes.y.minValue && y <= mRawPointerAxes.y.maxValue
-- 
2.17.1

